README.md

```diff
@@ -1,18 +1,38 @@
-# Governance
+# ESLint-Community Governance
 
-Outlines the governance of [@eslint-community](https://github.com/eslint-community) and fascilitates proposals and nominations
+## Purpose
 
-## Goal
+To have a place where community members can help ensure that widely depended upon ESLint-related packages live and never fall out of maintenance.
 
-As you can read in the
-["`eslint-community` GitHub organization" RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org),
-the goal of this organization is to have a place where community members can
-help ensure widely depended upon ESLint-related packages live and never fall out
-of maintenance.
+## Governance
+
+See [GOVERNANCE.md](./GOVERNANCE.md)
 
 ## The Community Core team
 
 * [`@aladdin-add`](https://github.com/aladdin-add)
 * [`@MichaelDeBoey`](https://github.com/MichaelDeBoey)
 * [`@ota-meshi`](https://github.com/ota-meshi)
 * [`@voxpelli`](https://github.com/voxpelli)
+
+
+## Projects
+
+### Primary projects
+
+* [eslint-plugin-es-x](https://github.com/eslint-community/eslint-plugin-es-x) (Collaborators: [@brettz9](https://github.com/brettz9))
+* [eslint-plugin-eslint-comments](https://github.com/eslint-community/eslint-plugin-eslint-comments) (Collaborators: )
+* [eslint-plugin-eslint-plugin](https://github.com/eslint-community/eslint-plugin-eslint-plugin) (Collaborators: [@bmish](https://github.com/bmish), [@bradzacher](https://github.com/bradzacher))
+* [eslint-plugin-n](https://github.com/eslint-community/eslint-plugin-n) (Collaborators: [@scagood](https://github.com/scagood))
+* [eslint-plugin-promise](https://github.com/eslint-community/eslint-plugin-promise) (Collaborators: [@xjamundx](https://github.com/xjamundx), [@bmish](https://github.com/bmish))
+* [eslint-plugin-security](https://github.com/eslint-community/eslint-plugin-security) (Collaborators: [@nzakas](https://github.com/nzakas), [@nlf](https://github.com/nlf))
+* [eslint-utils](https://github.com/eslint-community/eslint-utils) (Collaborators: )
+* [eslint-formatter-codeframe](https://github.com/eslint-community/) (Collaborators: )
+* [eslint-formatter-table](https://github.com/eslint-community/) (Collaborators: )
+* [eslint-plugin-mysticatea](https://github.com/eslint-community/) (Collaborators: )
+* [regexpp](https://github.com/eslint-community/) (Collaborators: )
+
+### Auxiliary projects
+
+* [eslint-stylistic](https://github.com/eslint-community/)  (Collaborators: [@Shinigami92](https://github.com/Shinigami92), [@antfu](https://github.com/antfu), [@kecrily](https://github.com/kecrily))
+  * [eslint-stylistic-repro-template](https://github.com/eslint-community/)
```

Comment:
I would like to open a follow up issue for this list afterwards and move eg. maybe the formatters to `auxiliary` + ensure that the list of maintainers are up to date and ensure that access to the repositories are synced up with that.